File: number_theory/fibonacci/javascript/fibonacci_number.js
/* n equals place in fib sequence
  send n to fib function to get the number from
  that position in the sequence
*/
function fib(n){
  let a = 1, b = 0, temp;

  while (n >= 0){
    temp = a;
    a = a + b;
    b = temp;
    n--;
  }

  return b;
}

File: number_theory/fibonacci/javascript/fibonacci_number.py
''' n equals place in fib sequence
  send n to fib function to get the number from
  that position in the sequence
'''

def fib(n):
    a = 0
    b = 1

    if n == 0:
        return a
    elif n == 1:
        return b
    else:
        for i in range(2,n):
            temp = a + b
            a = b
            b = temp
        return b


